ConsoleApp1/Calendar.cs

```diff
@@ -0,0 +1,244 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http.Json;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using static ConsoleApp1.Room;
+
+namespace ConsoleApp1
+{
+    internal class Calendar
```

Comment:
Where are the other projects? The homework is mainly about project references. 
You should separate business logic (validation and other operations) from UI (Console, prompting data) and from data (context, where you store data)